golang.md

```diff
@@ -0,0 +1,119 @@
+# Golang at Clever
+
+## Golang versions
+
+All golang services (with some exceptions) at Clever should be on go-1.5.
+
+
+## Docker Images
+
+#### drone
+
+`clever/drone-go:1.5`
+
+#### runtime
+
+  - `gliderlabs/alpine:3.2` - see [catapult/Makefile](https://github.com/Clever/catapult/blob/master/Makefile) as an example. (extra build configuration needed)
+  - `google/debian:wheezy`
+
+
+## Dependencies
```

Comment:
cc: @agclever @azylman @kvigen @ipam73 adding you as some Go people. Please add others who might have opinions about this. 

In case we are not sure of a decision right now, I think we should leave it the way it is until the guild can take over.